src/tooltip/Tooltip.js

```diff
@@ -271,7 +288,22 @@ class Tooltip extends React.PureComponent {
           onShow={onOpen}
           onRequestClose={onClose}
         >
-          {this.renderModalContent()}
+          {!withKeyboardAvoidView ? (
+            this.renderModalContent()
+          ) : (
+            <KeyboardAvoidingView
+              enabled={true}
+              keyboardVerticalOffset={StatusBar.currentHeight}
+              behavior={Platform.OS === 'android' ? undefined : 'position'}
```

Comment:
I think the behaviour should be `behavior={Platform.OS == "ios" ? "padding" : "height"}` i.e. "height" for `Android`
https://reactnative.dev/docs/keyboardavoidingview#behavior

Reply:
@pranshuchittora  : KeyboardAvoidView support in Modals was kind of little bit tricky . The default example syntax from the react-native docs was not working when used inside Modals as it is.

I tried multiple suggestions from the internet.
The solution which worked for our case was :  https://github.com/react-native-community/react-native-modal/issues/32#issuecomment-562524623

react-native's keyboardAvoidView implementation handles four cases "'height', 'position', 'padding,default" : 
https://github.com/facebook/react-native/blob/master/Libraries/Components/Keyboard/KeyboardAvoidingView.js

passing the behaviour as "undefined" will trigger out the default case in keyboardAvoidView
As for as android is concerned,  "default" setting  was working fine for our case.